config/initializers/csv_constants.rb

```diff
@@ -0,0 +1 @@
+CSV_CH3_TIMESERIES = 'Ch3_Fig3_Global_PA_Timeseries.csv'
```

Comment:
This is a good idea, I guess we might as well move the rest of the CSVs into this file as well when the time comes?